src/tools/data.js

```diff
@@ -14,6 +14,18 @@ const isPlainObj = o => {
 
 const comparison = (obj, needle) => obj === needle;
 
+const getObjectType = obj => {
+  if (_.isPlainObject(obj)) {
+    return 'Object';
+  }
+
+  if (_.isArray(obj)) {
```

Comment:
Likewise, native `Array.isArray` is preferred.

Reply:
Yep, same reason as above. Also don't feel strong about it here either.